satellite/templates/VM_info.html

```diff
@@ -37,12 +37,10 @@ <h2 class="card-pf-title">
                     </h2>
                 </div>
                 <div class="card-pf-body" id="vm_packages">
-                    <table class="table table-striped table-bordered table-hover" id="myPackages">
```

Comment:
@CURIOUSER005 This id is required to search packages in VM packages table. Without this searching won't work